compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -593,9 +593,13 @@ class ConvertNCHWToNHWC final : public luci::CircleNodeMutableVisitor<bool>
     }
     else if (multiplier == nullptr)
     {
```

Comment:
Can add checks if node->x() and y() are rank 4?

The transpose will be valid only for rank 4 inputs.